src/main/java/de/tum/in/www1/artemis/service/util/VCSSimulationUtils.java

```diff
@@ -0,0 +1,23 @@
+package de.tum.in.www1.artemis.service.util;
+
+import java.util.Random;
+
+import org.springframework.context.annotation.Profile;
+
+@Profile("dev")
+public class VCSSimulationUtils {
+
+    /**
+     * Simulates a commit Hash, the returned string consist out of 40 times the same number
+     * @return the simulated commitHash
+     */
+    public static String simulateCommitHash() {
+        Random random = new Random();
```

Comment:
Either use SecureRandom or UUID.randomUUID().toString()
You might need to replace "-" with a random number then.

Reply:
i added secureRandom